pkgs/top-level/all-packages.nix

```
@@ -28861,6 +28861,18 @@ with pkgs;
 
   picom = callPackage ../applications/window-managers/picom {};
 
+  picom-next = picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-next";
+    version = "vNext";
```

```suggestion
    version = "unstable-YYYY-MM-DD";
```

in accordance with https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

---

Resolved as indentation was fixed after moving the logic to it's own file